src/components/chat/unlockProfile/UnlockProfileWrapper.tsx

```diff
@@ -44,16 +60,15 @@ const UnlockProfileWrapper: FC<IntroContainerProps> = ({
             onClose={onClose}
           />
         </Modal>
-      ) : (
-        <Container className={type}>
-          <UnlockProfile
-            InnerComponentProps={{
-              type,
-              description,
-            }}
-            onClose={onClose}
-          />
-        </Container>
+      )}
+
+      {/* This is for displaying the Chat profile */}
+      {type === UNLOCK_PROFILE_TYPE.MODAL && position === MODAL_POSITION.ON_PARENT && (
```

Comment:
create a typescript type in commons.types.ts and keep the unlock profile modal type there.